packages/core/botpress/src/botpress.js

```diff
@@ -138,9 +138,7 @@ class botpress {
       }, 30 * 1000)
     }
 
-    // change the current working directory to botpress's installation path
-    // the bot's location is kept in this.projectLocation
-    process.chdir(path.join(__dirname, '../'))
+    global.botpressPath = path.join(__dirname, '../')
```

Comment:
instead of blobal can't you attach it to `bp` itself?

Reply:
@emirotin , I could try, but `bp` isn't available under `migrate.js` and `database/index.js` then.
Do you think it's better to pass it there?

Reply:
Yeah I generally think it's better to not use global if there's a clear way around

Reply:
@emirotin , done.